docs/src/pages/[platform]/theming/default-theme/colors/examples/ColorsExample.tsx

```diff
@@ -1,32 +1,26 @@
-import {
-  ThemeProvider,
-  Card,
-  Heading,
-  Text,
-  useTheme,
-} from '@aws-amplify/ui-react';
+import { Card, Heading, Text, ThemeProvider } from '@aws-amplify/ui-react';
 
-export const ColorsExample = () => {
-  const { tokens } = useTheme();
-  const theme = {
-    name: 'custom-theme',
-    tokens: {
-      components: {
-        card: {
-          backgroundColor: tokens.colors.background.secondary,
-          outlined: {
-            borderColor: tokens.colors.black,
-          },
-        },
-        text: {
-          color: tokens.colors.neutral[80],
-        },
-        heading: {
-          color: tokens.colors.brand.secondary[80],
+const theme = {
+  name: 'custom-theme',
+  tokens: {
+    components: {
+      card: {
+        backgroundColor: { value: '{colors.background.secondary}' },
+        outlined: {
+          borderColor: { value: '{colors.black}' },
         },
       },
+      heading: {
+        color: { value: '{colors.brand.secondary[80]}' },
+      },
+      text: {
+        color: { value: '{colors.brand.primary[80]}' },
```

Comment:
qq: when do we use `colors.brand.primary[80]`, when to use `colors.brand.primary.80`?